src/components/picker/helpers/usePickerMigrationWarnings.tsx

```diff
@@ -9,13 +9,13 @@ type UsePickerMigrationWarnings = Pick<
 >;
 
 const usePickerMigrationWarnings = (props: UsePickerMigrationWarnings) => {
-  const {children, migrate, getItemLabel, getItemValue, onShow} = props;
+  const {children, getItemLabel, getItemValue, onShow} = props;
   useEffect(() => {
     if (children) {
       LogService.warn(`UILib Picker will stop supporting the 'children' prop in the next major version, please pass 'items' prop instead`);
     }
 
-    if (migrate) {
+    if (Object.prototype.hasOwnProperty.call(props, 'migrate')) {
```

Comment:
This is not fixing the issue, when using `react-native-ui-lib` on another project I get this error:
```
Type '{ value: string; items: { value: string; label: string; }[]; migrate: boolean; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<PickerProps, any, any>> & Readonly<PickerProps>'.
  Property 'migrate' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<PickerProps, any, any>> & Readonly<PickerProps>'.
  ```
  
  snippet to reproduce:
  ```
   <Picker value={''} items={[{value: '1', label: 'one'}]} migrate />
   ```

Reply:
Let's revert this change and leave this file out of this PR.